lib/pages/LoginPage.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';

```

Comment:
this import fails as this folder doesn't exist. Please add the google_fonts folder. I think you forgot to git add it :)

Reply:
Have updated pubspec.yaml, should work now